test/benchgc/README.md

```diff
@@ -38,11 +53,25 @@ python -m benchgc [OPTIONS] --driver [DRIVER] --case [CASE]
 * if driver=pattern, please provide the pre-defined pattern name, such as mlp here
 * if driver is a dialect name, please provide the detail op name to start a single op test
 
+### --entry [str]
+* default : "entry"
+* the entry name of the kernel of input mlir or generated mlir
+
 ### --seed [int]
 * set the seed to generate the test data and reprodce the test
 
 ### --verbose [int]
-* set the verbose level
+* set the verbose level, default : 0
+* 0 : NO_VERBOSE
+* 1 : MODULE_VERBOSE, print the module will be executed
+* 2 : ARG_VERBOSE, + print arg information
+* 3 : COMPARE_VERBOSE, + print threshold for comparison
+* 4 : ERROR_OUTPUT_VERBOSE, + print all error data points if failed
+* 5 : OUTPUT_VERBOSE, + print all result including passed tensor
+* 6 : INPUT_VERBOSE, + print input torch tensors
```

Comment:
Do you think saving the tensor into a file will be better than printing them in the terminal?

Reply:
I agree with you, if the tensor is larger then dump into a file sounds better than printing. The printing thing is not added by this PR, I just added them on the `README.txt`, I can discuss with @WangJialei-A about, and maybe we could provide another option to dump. For this PR let's keep the printing.

Reply:
@ciyongch @xurui1995 
Need more discussion and design this part carefully.

Reply:
> @ciyongch @xurui1995 Need more discussion and design this part carefully.

For debug capability, we shall have a more convenient/flexible way to get the intermediate result.